app/src/test/java/com/onelist/ShoppingListUnitTest.kt

```diff
@@ -7,7 +7,7 @@ import org.junit.Test
 class ShoppingListUnitTest {
 
     @Test
-    fun shoppingListToString() {
+    fun `Given a shoppingList When the listID is 1, the name is My List, the length is 4 and the list is 1 and 2 Then the toString output is 1 - My list (4)`() {
```

Comment:
Naming unit tests with tick marks and the Given When Then. This is a good habit to start doing. https://www.javacodegeeks.com/2017/05/importance-given-unit-tests-tdd.html